client/my-sites/sidebar/sidebar.jsx

```diff
@@ -311,10 +311,6 @@ export class MySitesSidebar extends Component {
 			return null;
 		}
 
-		if ( site.jetpack ) {
```

Comment:
YES, let's remove this line (or add a safe guard) - it's causing annoying console errors when there's no site (loading phase, etc.)